.github/actions/spelling/action.yml

```diff
@@ -152,7 +152,7 @@ runs:
       # When updating the check-spelling action to a new release, update the 'cspell' prefix in the 'Merge dictionary
       # source prefixes' step with the dictionary version tag that belongs to the commit of the release. See
       # https://github.com/check-spelling/check-spelling/blob/v0.0.21/action.yml#L67 for current version.
```

Comment:
@Piedone The cspell prefix URL was not updated, see: https://github.com/check-spelling/check-spelling/releases/tag/v0.0.22

Reply:
Eh, I'm fixing that.

I'm pretty sure that when I checked out the release notes last week, it was much shorter, with a note along the lines of "you should be able to just point to this release without expecting breaking changes". Now it's full of stuff.

Reply:
Ouch. The release (notes) should receive its own release then. :D